.github/workflows/scripts/shared/verify_changes_update_version.sh

```diff
@@ -20,15 +20,20 @@ main() {
                                                                             -e 'test/'  \
                                                                             -e 'requirements-dev\.txt$' \
                                                                             -e '\.md$' \
-                                                                            $ADDITIONAL_DIFF_IGNORE)
+                                                                            $ADDITIONAL_DIFF_IGNORE \
```

Comment:
❓ Should we mention `ADDITIONAL_DIFF_IGNORE` in the the usage above?

Reply:
updated